app/models/incoming_email.rb

```diff
@@ -31,6 +31,8 @@ class IncomingEmail < ApplicationRecord
 
   belongs_to :client, touch: true
 
+  before_create { client.touch(:last_response_at) }
```

Comment:
This makes sense since after creating these incoming messages they are not updated (i.e. with status).